pkg/providers/tinkerbell/testdata/expected_results_cluster_tinkerbell_cp_external_etcd.yaml

```diff
@@ -22,10 +22,6 @@ spec:
     apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
     kind: TinkerbellCluster
     name: test
-  managedExternalEtcdRef:
```

Comment:
it would be less confusing to rename this file or remove the test case if external etcd doesn't apply

Reply:
It will apply eventually- this is temporary.